<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guanka</class>
 <widget class="QWidget" name="guanka">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>676</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>942</width>
    <height>676</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="level1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>level 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="level2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>level 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="level3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>level3</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
